libraries/AC_AutoTune/AC_AutoTune_Multi.h

```diff
@@ -0,0 +1,78 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  support for autotune of multirotors. Based on original autotune code from ArduCopter, written by Leonard Hall
+  Converted to a library by Andrew Tridgell
+ */
+
+#pragma once
+
+#include "AC_AutoTune.h"
+
+class AC_AutoTune_Multi : public AC_AutoTune
+{
+public:
+    // constructor
+    AC_AutoTune_Multi()
+    {
+        tune_seq[0] = 0;  // RD_UP
```

Comment:
these should be replaced with the actual enum values.

Reply:
got it.  Thanks!